tmessage/cli.py

```diff
@@ -35,6 +37,22 @@ def on_message(client, userdata, message):
     if user != current_user:
         print(Back.GREEN + Fore.BLACK + current_msg +
               Back.RESET + Fore.RESET + "")
+        store_messages(current_msg)
+
+
+folder_name = 'messages'
+session_start_date = datetime.now().strftime('%Y-%m-%d_%H:%M')
+
+
+def store_messages(raw_msg):

```

Comment:
The function will receive one more argument of `username` for storing received messages with default username being `current_user`.

Reply:
I changed the functionality of the store_messages so two arguments will not be needed
I have fixed it with one arg itself